site/docs/sql/00-intro.md

```diff
@@ -0,0 +1,141 @@
+---
+title: SQL Introduction
+description: Introduction to Hazelcast Jet SQL features.
+---
+
+Hazelcast Jet SQL service executes distributed SQL statements over
+Hazelcast collections and external data sets.
```

Comment:
This first statement blurs the difference between IMDG and Jet SQL a bit. There is only one collection supported at the moment - `IMap`.  How an end-user should decide which product to use? Maybe instead we may mention here that Jet SQL extends IMDG SQL, providing new use cases and features as described in the "Overview" section?